app/src/main/java/com/example/mycalculator/mvp/Operand.java

```diff
@@ -0,0 +1,47 @@
+package com.example.mycalculator.mvp;
+
+import static com.example.mycalculator.utils.StringUtils.EMPTY_STRING;
+
+public class Operand {
+
+    public String sign = EMPTY_STRING;
```

Comment:
you have already defined a EMPTY_STRING in StringUtils... please use that one

Reply:
correct. my bad.

Reply:
my badd